src/components/structures/RoomView.js

```diff
@@ -666,6 +683,24 @@ module.exports = React.createClass({
                 }
             }
 
+            // now we've decided whether or not to show this messages,
+            // add the read up to marker if appropriate
+            // doing this here means we implicitly do not show the marker
+            // if it's at the bottom
+            // NB. it would be better to decide where the read marker was going
+            // when the state changed rather than here in the render method, but
+            // this is where we decide what messages we show so it's the only
+            // place we know whether we're at the bottom or not.
+            var self = this;
+            if (prevEvent && prevEvent.getId() == this.state.readReceiptEventId) {
+                var hr;
+                hr = (<hr className="mx_RoomView_myReadMarker" style={{opacity: 1, width: '85%'}} ref={function(n) {
```

Comment:
Mostly because it starts intersecting with the timestamps & things at 100%. Obviously happy for you to tweak the CSS on this.